package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "homebridge-wiz-lan",
+  "name": "@chanconrad/homebridge-wiz-lan",
```

Comment:
I assume this is a typo

Reply:
Apologies - I thought this package had been abandoned so I made my fork a user scoped package so that a few other users could use it with this fix.

However, I accidentally made this commit onto this feature branch. I've now cherrypicked it on to my own main branch and reverted here, so I think this PR is ready if you're happy with it.

Thanks for looking into this!

Reply:
All good, to be fair I did basically abandon it lol